seq2science/rules/gene_counts.smk

```diff
@@ -3,11 +3,11 @@ if config["quantifier"] == "salmon":
     def get_counts(wildcards):
         quant_dirs = []
         if config["quantifier"] in ["salmon", "star"]:
-            for sample in treps[treps["assembly"] == wildcards.assembly].index:
+            for sample in treps[treps["assembly"] == ori_assembly(wildcards.assembly)].index:
```

Comment:
what does ori_assembly stand for? Original assembly?

Reply:
yup!